tests/api/users_test.py

```diff
@@ -31,7 +32,107 @@ def test_create_attributes(self):
             data='{"email": "test@example.net", "phone_number": null, "name": {"first_name": null, "middle_name": null, "last_name": null}, "create_user_as_pending": false, "attributes": {"ip_address": "203.0.113.1", "user_agent": "Toaster 3.0"}}',
             auth=client.auth,
             headers={
-                'Content-Type': 'application/json',
-                'User-Agent': f'Stytch Python v{version}'
+                "Content-Type": "application/json",
+                "User-Agent": f"Stytch Python v{version}",
             },
         )
+
+
+class TestSearchUsers(unittest.TestCase):
+    def test_search_no_params(self):
+        client = FakeClient()
+        response = FakeResponse(
+            status_code=200,
+            body='{"user_id": "user-test-183e939c-e7e1-4d56-82da-8ba9c6036878"}',
```

Comment:
I know this test doesn't use the response body, but this looks like the format for a different response. Either make it obviously not real (`"{}"` maybe) or use something that matches the search response format.